### PR TITLE
Add CWV target selector to page perf metric page

### PR DESCRIPTION
**Pull request by amortemousque**
Add the Core Web Vitals target selectors to the RUM browser monitoring page performance

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 1]
```diff
@@ -160,15 +159,21 @@ You can [enrich your RUM event data with feature flags][6] to get additional con
...
 | `view.first_input_delay`        | number (ns) | Time elapsed between a user's first interaction with the page and the browser's response.                                                                                                                             |
+| `view.first_input_delay_target_selector`      | string (CSS selector) | CSS selector of the first element the user interacted with.                                                                                                                |
+| `view.interaction_to_next_paint`| number (ns) | Longest duration between a user's interaction with the page and the next paint.                                                                                                                              |
+| `view.interaction_to_next_paint_target_selector`| string (CSS selector) | CSS selector of the element associated to the longest interaction to the next paint.                                                                                                          |
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/browser/data_collected.md:174]: small tweak -- "associated **with**"

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 2]
```diff
@@ -53,16 +53,30 @@ You can access performance metrics for your views in:
 | [Cumulative Layout Shift][9]  | Visual stability | Quantifies unexpected page movement due to dynamically loaded content (for example, third-party ads) where 0 means that no shifts are happening. | <0.1        |
 | [Interaction To Next Paint][19]| Interactivity    | Longest duration between a user's interaction with the page and the next paint. Requires RUM SDK v5.1.0. | <200ms        |
 
+### Core web vitals target elements
+
+Identifying what element triggered a high Core Web Vitals metrics is the first step in understanding the root cause and being able to improve performance.
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:58]: small tweak "Core Web Vitals **metric**" (singular)

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 3]
```diff
@@ -53,16 +53,30 @@ You can access performance metrics for your views in:
...
+RUM reports the element that is associated with each Core Web Vital instance:
+
+- For Largest Contentful Paint, we report the CSS Selector of the element corresponding to the largest contentful paint.
+- For Interaction to Next Paint, we report the CSS selector of the element associated to the longest interaction to the next paint.
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:62]: "associated **with**"

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 4]
```diff
@@ -53,16 +53,30 @@ You can access performance metrics for your views in:
...
 | `view.first_input_delay`        | number (ns) | Time elapsed between a user's first interaction with the page and the browser's response.                                                                                                                                        |
+| `view.first_input_delay_target_selector`      | string (CSS selector) | CSS selector of the first element the user interacted with.                                                                                                                |
 | `view.interaction_to_next_paint`| number (ns) | Longest duration between a user's interaction with the page and the next paint.                                                                                                                              |
+| `view.interaction_to_next_paint_target_selector`| string (CSS selector) | CSS selector of the element associated to the longest interaction to the next paint.                                                                                                          |
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:77]: "associated **with**"

**Review by nhientrandatadog — APPROVED** [review]
left a few wording comments, otherwise looks good!

**Review by alai97 — APPROVED** [review]

**Pull request merged**